pkg/cmd/checkendpoints/controller/pod_network_connectivity_check_controller.go

```diff
@@ -67,6 +74,16 @@ func NewPodNetworkConnectivityCheckController(podName, podNamespace string,
 // Sync ensures that the status updaters for each PodNetworkConnectivityCheck is started
 // and then performs each check.
 func (c *controller) Sync(ctx context.Context, syncContext factory.SyncContext) error {
+	// if PNCC CRD is not installed and established, cleanup any existing checkers, block
+	established, err := c.podNetworkConnectivityCheckResourceDefinitionEstablished()
```

Comment:
Also, this is in the wrong spot.  Placing it here means we'll have reflectors throwing perma-fails in our logs.  Do this check well in advance of starting the controllers.

Reply:
Updated.